resources/benchmark-runner.mjs

```diff
@@ -315,12 +334,35 @@ class RAFTestInvoker extends TestInvoker {
     }
 }
 
+class AsyncRAFTestInvoker extends BaseRAFTestInvoker {
+    _scheduleCallbacks(resolve) {
+        requestAnimationFrame(async () => {
+            await this._syncCallback();
+            requestAnimationFrame(() => {
+                setTimeout(() => {
```

Comment:
This means that there might or might not be another rAF tick before the timer runs, so this would bring back the issues we had with sp2's runner.